src/app/loadout-drawer/loadout-drawer-reducer.ts

```diff
@@ -180,38 +182,6 @@ export function addItem(
   });
 }
 
-/**
- * Defines the multilayered functionality for dropping an item in the loadout drawer.
- *
- * It does the following
- * 1. If the item is a subclass, it clears the currently selected subclass.
- *
- */
-export function dropItem(
```

Comment:
The "special features" of `dropItem` were already built in to `addItem` (though they likely were not when it was written).